### ISSUE TITLE
Ignore rules set via flag

### ISSUE THREAD

**Issue by Integralist** [mgechev/revive#738]
**Is your feature request related to a problem? Please describe.**
I have my editor setup to run revive on the current file. Revive will report (as an example) `should have a package comment` but if I run revive across the whole project it's fine because revive will see another file under the same package has a comment. The downside to running revive across the whole project is that it will report errors on files I'm not working on/responsible for and don't want to be distracted by.

**Describe the solution you'd like**
I'd like to setup my editor to run revive against the current file but able to set a flag to ignore specific rules that would fail when run against a single file but pass when run against the whole project (e.g. `--ignore rule_x --ignore rule_y`).

**Describe alternatives you've considered**
To explain what else I've tried I need to demonstrate my editor configuration:

```lua
    vim.cmd([[
      fun! GoLint()
       :silent !gofumpt -w %
       :edit
       :cex system('revive '..expand('%:p')) | cwindow
     endfun
     autocmd! BufWritePost *.go call GoLint()
    ]])
```

I can change `%:p` to `%:p:h` which will strip the current file's name leaving an absolute path to the package directory which means the problem is avoided, but it's a similar issue to running revive globally that I might see results from other files in the package that I'm not working on and don't want to be distracted by (albeit slightly less distracting).

**Comment by chavacava** [comment]
Hi @Integralist, thanks for filling the issue.
You can enable/disable rules through the configuration file. You can setup your IDE to run `revive` with the configuration file of your convenience.
